Content.Server/Shuttles/Systems/ShuttleSystem.Impact.cs

```diff
@@ -54,5 +59,8 @@ private void OnShuttleCollide(EntityUid uid, ShuttleComponent component, ref Sta
         var audioParams = AudioParams.Default.WithVariation(0.05f).WithVolume(volume);
 
         _audio.Play(_shuttleImpactSound, Filter.Pvs(coordinates, rangeMultiplier: 4f, entityMan: EntityManager), coordinates, true, audioParams);
+        
+        var kineticEnergy = ourBody.Mass * Math.Pow(jungleDiff, 2) / 2;
```

Comment:
```suggestion
        var kineticEnergy = ourBody.Mass * Math.Pow(ourVelocity.Length, 2) / 2;
```
Это же формула кинетической энергии, почему мы возводим в квадрат скорость разницы двух тел, а не скорость нашего тела?